dataverse/orgsvc/C#-NETCore/AttachmentAndAnnotationOperations/README.md

```diff
@@ -37,7 +37,8 @@ When the sample runs, you will be prompted in the default browser to select an e
 }
 ```
 
->**Tip**: You can set a user environment variable named DATAVERSE_APPSETTINGS to the file path of the appsettings.json file stored anywhere on your computer. The samples will use that appsettings file if the environment variable exists and is not null. Be sure to log out and back in again after you define the variable for it to take affect. To set an environment variable, go to **Settings > System > About**, select **Advanced system settings**, and then choose **Environment variables**.
+> [!Tip]
```

Comment:
@w-azure Why did you remove the instructions on how to set an environment variable?